test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -11069,6 +11077,16 @@ def setup_sched_priv(self, sched_priv=None):
         self.holidays_obj = Holidays()
         self.holidays_parse_file(level=logging.DEBUG)
 
+    def setup_sched_logs(self):
+        if 'sched_log' in self.attributes:
+            sched_logs = self.attributes['sched_log']
+        else:
+            sched_logs = os.path.join(self.pbs_conf['PBS_HOME'],
+                                      'sched_logs')
+
```

Comment:
Check if the path exists first like setup_sched_priv()

Reply:
setup_sched_priv only checks the path before parsing the config, we don't do that.